### DISCUSSION TITLE
快速切换目标语言和翻译语言

### DISCUSSION THREAD

**Discussion by happykkAi** [pot-app/pot-desktop#277] [Ideas]
我有时候需要打开面板输入中文翻译成英文，都需要选择翻译的语言，能不能在图标的位置有个互相转换的翻译语言的按钮 （后台可以设置好翻译语言）
<img width="426" alt="iShot_2023-06-28_00 11 42" src="https://github.com/pot-app/pot-desktop/assets/92377310/ef329e1f-b056-4ba2-b58c-07f573640f1b">

**Comment by Pylogmon** [comment]
现在就可以呀，第二目标语言，你设置为英文就行，

直接翻译就行，不需要切换语言

**Reply by happykkAi** [reply]
我第二语言设置为英文  是不是比如我输入中文 （默认目标语言为中文的情况下）会自动翻译成英文吗？（还是需要下拉选择语言）
<img width="433" alt="image" src="https:github.com/pot-app/pot-desktop/assets/92377310/b2e26a87-29b9-4173-95e9-b20b47d8371a">

**Reply by happykkAi** [reply]
<img width="433" alt="image" src="https://github.com/pot-app/pot-desktop/assets/92377310/c1e7dfda-8173-4415-9568-3c3e1a0a1d7b">

**Comment by happykkAi** [comment]
我理解了 我测试了一下效果  腾讯能够正常翻译 
<img width="423" alt="image" src="https://github.com/pot-app/pot-desktop/assets/92377310/d0c620ae-29ae-486b-a69f-7d1a2e3b3a27">
